coding-concepts.js

```diff
@@ -15,45 +15,45 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 
 // --------------------1) What will this log?
 
-const cohort = "LEARN 2022"
-// console.log(cohort.length)
+//const cohort = "LEARN 2022"
+//console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: This will log the amount of text in the constant.
+// b) Verify and explain: This logged 10 the amount of text in the constant.
 
 
 // --------------------2) What will this log?
 
-const greeting = "Hello World!"
-// console.log(greeting[4])
+//const greeting = "Hello World!"
+//console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: This will log the letter o in Hello.
+// b) Verify and explain: This logged the letter o in Hello.
```

Comment:
For the verify and explain section, please expand a little further as to why it is logging that specifically.